hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/StorageVolume.java

```diff
@@ -75,6 +75,7 @@ public abstract class StorageVolume
   // The name of the directory where temporary files used to check disk
   // health are written to. This will go inside the tmp directory.
   public static final String TMP_DISK_CHECK_DIR_NAME = "disk-check";
+  public static final int HUNDRED_MB = 100 * 1024 * 1024;
```

Comment:
If we end up using this value, it would be better represented as `100 * OzoneConsts.MB`.